README.md

```diff
@@ -34,7 +34,7 @@ We are planning to use the following tools:
 * [Postgres](https://www.postgresql.org) with [TypeORM](https://typeorm.io/#/)
 * [Next.js](https://nextjs.org/) for both client and server-side rendering of the frontend (NextJS is based on [React](https://reactjs.org))
   * [JavaScript/TypeScript](https://www.typescriptlang.org/index.html#download-links)
-  * [Styled Components](https://www.styled-components.com) for styling.
+  * [Material UI](https://material-ui.com/) for components and it's built in `makeStyles` hook and `styled` HOC for custom styling
```

Comment:
```suggestion
  * [Material UI](https://material-ui.com/) for components and its built-in `makeStyles` hook and `styled` HOC for custom styling
```
Also, what is an HOC?

Reply:
Thx for the typo fix.
HOC or higher order components is a pattern where you wrap a component in a function

```jsx
const StyledGrid = styled(Grid)({
  bacgroundColor: "red"
});
```

I specifically mentioned that because I myself didn't know material UI supported this, and this is a main way you do styling with styled components.